import_map_importer_import_meta_url_test.ts

```diff
@@ -8,7 +8,7 @@ import type { ImportMap } from "./import_map.ts";
 afterAll(async () => {
   try {
     const cacheDir = fromFileUrl(
-      new URL("./.test_cache", import.meta.url),
+      new URL("./.test_cache_meta", import.meta.url),
```

Comment:
We need to ignore `./test_cache_meta` in `deno.jsonc` like `./test_cache`.

https://github.com/lambdalisue/deno-import-map-importer/blob/1cc27e732c357b3f124cd38d31d7ad9955f93255/deno.jsonc#L31

Reply:
Fixed. Note that the `--exclude` option requires a backslash for dot matching as it uses regex patterns.